plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -16,6 +16,7 @@ public class AndroidCommandPluginExtension {
     private final String androidHome
     private final MonkeySpec monkey
     private final NamedDomainObjectContainer<InputExtension> scriptsContainer
+    final NamedDomainObjectContainer<DemoModeExtension> demoModeContainer
```

Comment:
Does this one need more visibility than the one above?

Reply:
I was exposing a public getter already for other properties. But instead, I decided to just make those public since there is no harm and will make the extension simpler.

Reply:
Will update the others in a follow-up to sync.